handlers/graph.go

```diff
@@ -632,15 +633,20 @@ func graphNode(w http.ResponseWriter, r *http.Request, client *prometheus.Client
 		checkError(errors.New(fmt.Sprintf("Node graph does not support the 'namespaces' query parameter or the 'all' namespace")))
 	}
 
-	namespace := o.Namespaces[0]
-	n := graph.NewNode(namespace, o.NodeOptions.Workload, o.NodeOptions.App, o.NodeOptions.Version, o.NodeOptions.Service, o.GraphType)
+	// Here, it's true that o.Namespaces has only one item. So, it's safe to use "for" knowing
+	// that only one iteration will happen.
+	var n graph.Node
+	var namespace graph.NamespaceInfo
+	for _, namespace = range o.Namespaces {
+		n = graph.NewNode(namespace.Name, o.NodeOptions.Workload, o.NodeOptions.App, o.NodeOptions.Version, o.NodeOptions.Service, o.GraphType)
+	}
```

Comment:
Not a big deal, but why replace the 2 lines with 7?

Reply:
Since `o.Namespaces` is no longer an array, it's not possible to do namespace := o.Namespaces[0].
Perhaps, by removing the comments it will be only 5 lines?